include/util.h

```diff
@@ -14,10 +14,16 @@
 
 /* Some assemblers don't recognise the ul/ull (unsigned long) suffix */
 #define UL_CONST(x) x
+#define ULL_CONST(x) x
 
 #else /* not __ASSEMBLER__ */
 
-#define UL_CONST(x) PASTE(x, ul)
+/* There is no difference between using 'ul' or 'lu' as number postfix. Just
+ * when it comes to the printf() format specifiers, '%lu' is the only form that
+ * is supported.
+ */
+#define UL_CONST(x) PASTE(x, lu)
+#define ULL_CONST(x) PASTE(x, llu)
```

Comment:
I honestly don't think adding a layer of indirection around `llu` is improving code readability. This might be a convention that you are used to, but for me it's harder to read.

Reply:
I'm just following what already existed in the code and adapt the changes I brought in a while ago to this. I do not have any preference for one was or the other. The argument for these macros is, that constants defined with `UL_CONST(x)` or `ULL_CONST(x)` can also be used in the assembly code easily, as the macro is defined to just drop any postfix there. If we use `PASTE(x, something)` directly, there is no way to use such constants in assembly code,`PASTE()` is a helper macro for the preprocessor that is used in other places also and we can't define this to drop the `something` there.